lua/entities/gmod_wire_expression2/core/custom/tracesystem.lua

```diff
@@ -217,19 +217,14 @@ end
 
 local function RaySphereIntersection( Start, Dir, Pos, Radius )
 	local A = Dir:LengthSqr()
-	local R = Vector(Start)
-				R:Sub(Pos)
+  if(A == 0) then return nil end -- Div zero

```

Comment:
Need tab

Reply:
Fixed in: https://github.com/wiremod/wire-extras/pull/122/commits/2357f0fa69a3b98433934285d50a878025fe9223